cli.go

```diff
@@ -173,7 +176,21 @@ func (cli *CLI) parseFlags(args []string) (*Config, bool, bool, bool, error) {
 
 	// Deprecated options
 	var deprecatedSSLNoVerify bool
-	flags.BoolVar(&deprecatedSSLNoVerify, "ssl-no-verify", !config.SSL.Verify, "")
+	flags.BoolVar(&deprecatedSSLNoVerify, "ssl-no-verify", false, "")
+
+	// Because Go's default value for booleans are false, we need to track
+	// auxillary values to see if they were actually provided. It has a pretty
+	// high degree of sadness.
```

Comment:
@sethvargo can't we just use the `true` default value for the flag, and then merge the other configs against the CLI config?